pmd-core/src/main/ant/alljavacc.xml

```diff
@@ -71,6 +71,10 @@
         <replace file="${target}/net/sourceforge/pmd/lang/ast/dummy/JavaCharStream.java"
                  token="throw new Error(t.getMessage())"
                  value="throw t" />
+        <!-- Set tab size of JavaCC to 1 -->
+        <replace file="${target}/net/sourceforge/pmd/lang/ast/dummy/JavaCharStream.java"
```

Comment:
Alternatively there is also a setter, we could call. But for now, I think, it's ok let it hard code to 1. We could add later a configuration option, to use a custom tab size, see #292

Reply:
I originally wanted to use the setter, but that requires introducing a subclass just to call the getter (since it's protected). I think this is the cleaner solution for now, but that changes when introducing a configuration option for it, of course.